cmd/kubeadm/app/util/version.go

```diff
@@ -25,9 +25,10 @@ import (
 )
 
 var (
-	kubeReleaseBucketURL  = "https://storage.googleapis.com/kubernetes-release/release"
+	kubeReleaseBucketURL  = "https://dl.k8s.io"
 	kubeReleaseRegex      = regexp.MustCompile(`^v?(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)([-0-9a-zA-Z_\.+]*)?$`)
 	kubeReleaseLabelRegex = regexp.MustCompile(`^[[:lower:]]+(-[-\w_\.]+)?$`)
+	kubeBucketPrefixes    = regexp.MustCompile(`^((release|ci|ci-cross)/)?([-\w_\.+]+)$`)
```

Comment:
We will only allow `release` and `ci`. What is indeed a little bit confusing is that we're gonna use artifacts from `ci-cross` under the hood (kubeadm user uses `ci/latest` => will actually fetch `ci-cross/latest.txt` for the version to use)

This is due to that the normal ci job does not push images